web.rb

```diff
@@ -397,6 +398,7 @@ def initialize(user_id, expansions, singletons)
         self['user_id'] = user_id
         self['expansions'] = expansions
         self['singletons'] = singletons
+        self['checks'] = checks
```

Comment:
`checks` isn't defined.  Like `expansions` and `singletons` it will need to be passed in as an argument to `Collections.new()` which is called in `get_collection()` (line 108) where you should set a sensible default for a brand new collection, as well as line 403 where you handle the case where you load a collection that may not already have a `checks` key (basically, all collections right now).  To match the existing behavior, add another `nil` added to match the new initializer signature, and on your end you should have some logic to deal with getting `null` back in that case.